File: python/simple-iteration.py
import math
X0 = float(input('Enter value for Xo: '))
iteration = int(input('Enter number of iterations: '))
for n in range(1, (iteration + 1)):
    X0 = round(math.cos(X0), 10)
    print('X' + str(n) + ' = cosX' + str(n-1) + ' = '+ str(X0))

File: java/SimpleIteration.java
package java;

class SimpleIteration {

    private static void println(Object object) {
        System.out.println(object);
    }
    
    private static void print(Object object) {
        System.out.print(object);
    }

    private static double round(double value) {
        double result = Math.round((value * 1000000000) / 1000000000D);
        return result;
    }

    public static void main(String[] args) {
        java.util.Scanner scan = new java.util.Scanner(System.in);

        print("Enter value for Xo: ");
        double Xo = scan.nextDouble();

        for(int n=1; n < 12; n++) {
            Xo = round(Math.cos(Xo));
            println("X" + n + " = cosX" + (n-1) + " = " + Xo);
        }
    }
}